apps/ios/package.json

```diff
@@ -13,7 +13,8 @@
     "lint": "fluentui-scripts eslint",
     "start": "react-native start",
     "bundle": "fluentui-scripts metro --cli",
-    "bundle-dev": "fluentui-scripts metro --dev --cli"
+    "bundle-dev": "fluentui-scripts metro --dev --cli",
+    "ios": "react-native run-ios --project-path src & react-native start"
```

Comment:
This looks like it runs it in the same terminal instance. For it to fix CI, it'll need to kick it off in a different window. The way CI runs is it has one terminal window open and runs each command sequentially so launching the packager would then not let any other commands execute

Reply:
Noted. I could also have the packager run in the background instead of run-ios, but I think that is a less preferable approach. I'll look for a way to launch a new window

Reply:
Opening up a new terminal window was the complication we fixed in react-native-macos I explained over teams. That's why it got a bit convoluted to run the packager